replay/models/ucb.py

```diff
@@ -0,0 +1,114 @@
+from typing import Optional
+
+import pandas as pd
+
+from pyspark.sql import DataFrame, Window
+from pyspark.sql import functions as sf
+
+from replay.models.base_rec import Recommender
+
+
+class UCB(Recommender):
+    """
+    Calculates upper confidence bound (UCB) for the confidence interval
+    of true fraction of positive ratings.
+
+    ``relevance`` must be converted to binary 0-1 form.
```

Comment:
add calculation formula and explanation why you fillna with 1

Reply:
because you need to fill new items relevance. but if you use formulae for ucb you get uncertainty.

Reply:
I meant why is it equal to one. Does not matter any more